README.md

```diff
@@ -42,7 +42,12 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+
+for (var i = 0; i < 2; i++) {
+  for (var j = 0; j < 3; j++) {
+  console.log("I want to eat a "+foods[i][j])
+  }
+}
 ```
```

Comment:
✅  though I would recommend setting the condition using the `length`s of each array instead of hard-coded values